openspec/changes/add-emilkowalski-agent-skills/tasks.md

```
@@ -0,0 +1,24 @@
+## 1. Canonical skills and provenance
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)